omniwitness/omniwitness.go

```diff
@@ -269,6 +235,61 @@ func runDistributors(ctx context.Context, c *http.Client, g *errgroup.Group, log
 	})
 }
 
+// Feeder is an enum of the known feeder types.
+type Feeder uint8
+
+const (
+	Serverless Feeder = iota + 1
+	SumDB
+	Pixel
+	Rekor
+)
+
+var (
+	feederByName = map[string]Feeder{
+		"serverless": 1,
```

Comment:
Maybe this would be better as:

```go
"serverless": Serverless,
"sumdb":      SumDB,
...
```

it'd be very easy to spot any errors in the mapping down the line.